### ISSUE TITLE
Interactive web tool to pick color names and RGBs of GMT supported colors

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#7875]
**TL;DR** A web tool (https://seisman.github.io/gmt-color-picker/) to interactively pick GMT's color name/RGB

### Motivation

GMT supports more than 600 color names. On this page (https://docs.generic-mapping-tools.org/6.4/gmtcolors.html), we provide two different ways to show all the supported colors:

1. The image [GMT_RGBchart.png](https://docs.generic-mapping-tools.org/latest/_images/GMT_RGBchart.png)
2. The big/long table

They both have some shortcomings. 

For the image, it's easy to visually find the color I like, but I can't copy the color's name or RGB value. So I have to manually type the color name and sometimes I make typos.

For the big table, I can easily copy the color name/RGB, but I can't visually see the color from the table.

### Solution

With the help of ChatGPT, I've created a web tool on which we can interactively pick the color we like and get the color name/RGB. Actually, I just told ChatGPT what I need, and ChatGPT wrote all the codes. The web tool is written in HTML + CSS + Javascript, so it's quite lightweight. 

Simple features:

- Place your mouse over the color box to see the color name/RGB
- Click the color box to copy the color name/RGB to the clipboard

**URL:** https://seisman.github.io/gmt-color-picker/

You can visit the above site to try it. If you're interested in the source code, here is the link: https://github.com/seisman/gmt-color-picker.

### Preview

<img width="1193" alt="image" src="https://github.com/GenericMappingTools/gmt/assets/3974108/1b8edb97-72a9-428c-8934-d40c458c1b87">

### What's next?

I think there are many possible options:

1. Keep it as a standalone web tool and move the repository from my personal account to the GenericMappingTools organization. [The URL will be https://generic-mapping-tools.org/gmt-color-picker]
2. Include it into the GMT docs as a standalone page [The URL will be https://docs.generic-mapping-tools.org/latest/color-picker]
3. Embed the web tool on the [gmtcolors page](https://docs.generic-mapping-tools.org/latest/gmtcolors.html) and remove the big table [I think it's doable]
4. Keep the repository under my own GitHub account and link to the tool in the GMT docs

Comments?

**Comment by joa-quim** [comment]
This is nice (and useful). My favorite solution is **2.** followed with a ``docs`` adding such that we can do ``gmt docs colors``

**Comment by PaulWessel** [comment]
I think 2 as well but up to @seisman i think

**Comment by seisman** [comment]
I think we will go with option 2.

Ping @GenericMappingTools/gmt-docs for thoughts about how this tool can be improved before including it in the GMT docs.

**Comment by PaulWessel** [comment]
Seems to be very functional as is.  I suggest we include it as is and once 6.5 is released and the masses finds it we are sure to get some feedback for a revision.
